addons/account/models/sequence_mixin.py

```diff
@@ -71,6 +71,9 @@ def _get_starting_sequence(self):
         self.ensure_one()
         return "00000000"
 
+    def _get_highest_query(self):
+        return "SELECT {field} FROM {table} {where_string} ORDER BY {field} DESC LIMIT 1 FOR UPDATE"
+
```

Comment:
Why did it was needed to extract the query in its own method?
I don't find any overwrite.
What kind of overwrite is needed?

Reply:
It was for cases where e.g. you can not take into account the first letters in the order by.  But it is indeed not inherited @jjscarafia @Danisan @zaoral

Reply:
The `order by` is wrong. It should use the regex (see #52790).
I don't think it needs to be overwritable.

Reply:
Hi @KangOl and @jco-odoo

As far as I remember, this was made because we needed to extend the logic to search highest for account.move that use documents (logic in l10n_latam_invoice_documents) and l10n_ar and l10n_cl localizations.

I think that in the development process this was not neccesary anymore. I have tested in master and saas-13.3 and is working rigth now.

So we can remove this method and work with the query as it was before the change